MonkeyLoader/MonkeyLoader.csproj

```diff
@@ -25,6 +25,7 @@
     <EmbedAllSources>True</EmbedAllSources>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddSymbolsToPackage</TargetsForTfmSpecificContentInPackage>
+    <UserSecretsId>9d268757-c7c1-44c8-9534-86ab81c5fc15</UserSecretsId>
```

Comment:
Is this supposed to be here?

Reply:
Ah, no, guess VS did a funny when I accidentally clicked on "Manage User Secrets" in the project context menu